apps/web/components/ui/form/CheckedSelect.tsx

```diff
@@ -95,6 +69,4 @@ export const CheckedSelect = (props: CheckedSelectProps) => {
   );
 };
 
-CheckedSelect.displayName = "CheckedSelect";
```

Comment:
This was probably a remnant of a once existing forwardRef which has been removed since.